app/misc/authenticate.ts

```diff
@@ -139,7 +139,7 @@ function getUserInfo(callback) {
           $("#otpModal").modal('show');
         });
       }
-      else if (err == "Error: getaddrinfo ENOTFOUND api.github.com api.github.com:443"){
+      else if (err.toString().indexOf("ENOTFOUND") != 0 || err.toString().indexOf("ENOENT") != 0){
```

Comment:
It would be nice if we could somehow check err.errno or something similar? If not possible, perhaps we should make it a TODO. Let me know the results?

Performing multiple string comparisons is inefficient :(; however, it looks like this is what others have done in this file before.

Reply:
err.errno I think will work! Can you tell me what it returns on mac? I think it'll still require the multiple string comparisons but we can at least get rid of the indexof operation.